Common Programming Concepts/Data Types/Arrays/tests/tests.rs

```diff
@@ -1,11 +1,12 @@
+use arrays::arr;
+
 #[test]
 fn if_compiles() {
     //Mocking test to run only the compiler's checker
     assert!(true);
 }
 
-
-
-
-
-
+#[test]
+fn test_arr() {
+    assert_eq!(arr(), "Wow, that's a big array!")
```

Comment:
Let's mention the output we do expect in the task description for the learner not to modify it